scripts/import/laptops/selfreport2csv

```diff
@@ -171,7 +172,14 @@ def load_data(
         if verbose:
             print("redcap.export_records lacks a date_begin param, skipping")
         data = _load_data()
-
+    
+    # Turning time data into correct date-time
+    data['youth_report_2_timestamp'] = pd.to_datetime(data['youth_report_2_timestamp'], format='%Y-%m-%d %H:%M:%S', errors='coerce')
+    for i in range(0, data.shape[0]):
```

Comment:
This should be doable without the for-loop:

```python
days_since_start = (datetime.now() - data['youth_report_2_timestamp']).days  # now an indexed vector of days
selected_idx = (data['youth_report_2_complete'] == '2') | (days_since_start > 30)
return data.loc[selected_idx]
```

Reply:
Fixed! Note that instead of `.days`, I decided to utilize `.astype('timedelta64[D]')`. Seemed to resolve the error that popped up when attempting to solve.

Reply:
Right! It should have been `.dt.days`, sorry.